aioconsole/execute.py

```diff
@@ -148,3 +148,31 @@ async def aexec(source, local=None, stream=None, filename="<aexec>"):
         if isinstance(tree, ast.Interactive):
             exec_single_result(result, new_local, stream)
         full_update(local, new_local)
+
+
+async def aeval(source, local=None):
+    """Asynchronous equivalent to *eval*."""
+    if local is None:
+        local = {}
+
+    if not isinstance(local, dict):
+        raise TypeError("globals must be a dict")
+
+    key = "__result__"
+    # Ensure that the result key is unique within the local namespace
+    # while key in local: key += "_"
```

Comment:
Why is this commented out?

Reply:
I thought it might be overkill 😅 But wanted to leave there for reference.  Are you in favor of remove that line or uncomment it?

Reply:
I decided to keep this logic [54e6c09](https://github.com/vxgmichel/aioconsole/pull/120/commits/54e6c0911733f2993dc33a59fb3758808d408a73)